applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h

```diff
@@ -55,6 +55,21 @@ enum bt_mgmt_scan_type {
 
 #define BRDCAST_ID_NOT_USED (BT_AUDIO_BROADCAST_ID_MAX + 1)
 
+/**
+ * @brief	Load advertising data into an advertising buffer.
+ *
+ * @param[out]     adv_buf         Pointer to the advertising  buffer to load.
+ * @param[in/out]  index           Next free index in the advertising buffer.
+ * @param[in]      adv_buf_vacant  Number of free advertising buffers.
+ * @param[in]      type            Type of the data.
+ * @param[in]      data_len        Length of the data.
```

Comment:
Mismatch between the order in the doc and the function. These two should swap place

Reply:
Swopped these round so many times I think I got lost!